### PR TITLE
Adopt to schema

### PR DESCRIPTION
**Pull request by DunklesArchipel**
Adopt the YAML files to version 0.1.0 of the echemdb MDS

### CODE REVIEW

File: literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4d_red.yaml [hunk 1]
```diff
@@ -12,23 +12,25 @@ system:
...
           value: 7.5
           unit: mol L⁻¹
-    pH:
-      estimation:
-        value: -1.5
-        calculation: CurTiPot
+    ph:
+      value: -1.5
+      calculation: CurTiPot
+      comment: estimated
```
Comment (linuxrider) [inline literature/sato_2006_effect_725/sato_2006_effect_725_p3_f4d_red.yaml:33]:
```suggestion
    ph:
      value: -1.468
      temperature: 
          value: 25
          unit: °C
      comment: estimated with CurTiPot on extraction from publication
```

File: literature/kibler_2000_in-situ_73/kibler_2000_in-situ_73_f8a_black.yaml [hunk 2]
```diff
@@ -14,24 +14,25 @@ system:
...
-        calculation: CurTiPot
+    ph:
+      value: 0.1
+      calculation: CurTiPot
+      comment: estimated
     temperature:
       value: 20
       unit: C
-      comment: room temperature
+    comment: room temperature
```
Comment (linuxrider) [inline literature/kibler_2000_in-situ_73/kibler_2000_in-situ_73_f8a_black.yaml:35]:
```suggestion
      comment: room temperature
```

File: literature/kibler_2000_in-situ_73/kibler_2000_in-situ_73_f7a_black.yaml [hunk 3]
```diff
@@ -14,24 +14,25 @@ system:
...
-        calculation: CurTiPot
+    ph:
+      value: 0.1
+      calculation: CurTiPot
+      comment: estimated
     temperature:
       value: 20
       unit: C
-      comment: room temperature
+    comment: room temperature
```
Comment (linuxrider) [inline literature/kibler_2000_in-situ_73/kibler_2000_in-situ_73_f7a_black.yaml:35]:
```suggestion
      comment: room temperature
```

**Review by linuxrider — COMMENTED** [review]
Fixed wrong pH values.

**Review by linuxrider — APPROVED** [review]

**Comment by DunklesArchipel** [comment]
@linuxrider I see your point in these changes. At the moment they are not compatible with the schema. 
* Temperature would would be quantitiy property in the schema, which only allows unit and value. This is probably not enough. Instead it should have also a dimension, comment or alterantive name. 
* the pH schema also does not allow your changes. I will adjust the pHs and add the other information in a comment of pH or the comment of the electrolyte. We address this in the next version of the database. I will open an issue once the data has been migrated.

**Pull request merged**